### PR TITLE
feat: allow .ggp preview fetching

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Make web request previews for .ggp files.

## Related Issue
- Works towards https://github.com/owncloud/enterprise/issues/6907

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

**Comment by kulmann** [comment]
Waiting for an answer on https://github.com/owncloud/ocis/pull/10304#issuecomment-2444050588 before this gets merged.

**Review by AlexAndBear — APPROVED** [review]

**Pull request merged**
